### ISSUE TITLE
Flink Kinesis connector available in Maven Central

### ISSUE THREAD

**Issue by apaleyes** [amzn/milan#12]
It appears as though the connector is now available: https://search.maven.org/artifact/org.apache.flink/flink-connector-kinesis_2.12

Does this make the "Environment Setup" section in readme obsolete?

**Comment by tborchert** [comment]
Probably, have you tried updating it to use the version from maven central?

**Comment by apaleyes** [comment]
Nope, haven't got around to try it yet. If i did, that would have been a PR!
